sass/blocks/main-nav/_main-nav.scss

```diff
@@ -74,4 +75,73 @@
     display: inline-block;
     text-decoration: none;
   }
+
+  // Tablet screen
+  @include screen-size($tablet-size) {
+    &__list {
+      display: flex;
+      background-color: $color-white;
+    }
+
+    &__item {
+      padding: 14px 0 17px;
+      width: 10rem;
+      margin-left: 7%;
+      border: none;
+
+      &:first-child {
+        margin-left: 3.5%;
```

Comment:
you're adding left margin to the first item and same margin to the last item to have some padding around, I think better way would be to add a wrapper that sets those other than elements themselves do it.
same as you have with any other __inner class on the page.